frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Runs.feature

```diff
@@ -97,43 +128,58 @@ Scenario Outline: Download the logs from Pipeline Details page : P-04-TC05
 
 
 @regression
-Scenario Outline: kebab menu options in pipelines page : P-04-TC07
-    Given user is at the Pipelines page
+Scenario: kebab menu options in pipelines page : P-04-TC07
+    Given user is at pipelines page
 
 
 @regression
-Scenario Outline: Start LastRun from topolgy page : P-05- TC04
-    Given user is at the topolgy page
+Scenario: Start LastRun from topolgy page : P-05- TC04
+    Given user is at the Topolgy page
     And one pipeline run is completed with the workload
 
 
 @regression
-Scenario Outline: Start LastRun from topolgy page : P-05- TC04
-    Given user is at the topolgy page
+Scenario: Start LastRun from topolgy page : P-05- TC04
+    Given user is at the Topolgy page
     And one pipeline run is completed with the workload
 
 
-@regression, @smoke
-Scenario Outline: Maximum pipeline runs display in topology page: P-05-TC05
-    Given user is at the topolgy page
-    And 5 pipeline runs are completed with the workload
-
+@regression
+Scenario: Maximum pipeline runs display in topology page: P-05-TC05
+    Given 5 pipeline runs are completed with the git workload
+    And user is at the Topolgy page
+    When user clicks on the node name
+    Then side bar is displayed with the pipelines section
+    And 3 pipeline runs are displayed under pipelines section of topolgy page
+ 
 
 @regression, @manual
-Scenario Outline: Download the logs from Pipeline Details page after pipleine run: P-05-TC06
+Scenario: Download the logs from Pipeline Details page after pipleine run: P-05-TC06
     Given user is at the Pipeline Details page
 
 
-Scenario Outline: Start the pipeline wtih cancelled tasks: P-07- TC04
+Scenario: Start the pipeline with cancelled tasks: P-07- TC04
     Given user is at the Pipeline Details page
-    And pi[peline run is available with cancelled tasks
+    And pipeline run is available with cancelled tasks
 
 
-Scenario Outline: Start the pipeline wtih failed tasks: P-07- TC05
+Scenario: Start the pipeline with failed tasks: P-07- TC05
     Given user is at the Pipeline Details page
-    And pi[peline run is available with failed tasks
+    And pipeline run is available with failed tasks
 
 
-Scenario Outline: Start the pipeline wtih successful tasks: P-07- TC06
+Scenario: Start the pipeline with successful tasks: P-07- TC06
     Given user is at the Pipeline Details page
-    And pi[peline run is available with failed tasks
+    And pipeline run is available with failed tasks
+
+
+@regression, @smoke
+Scenario Outline: Pipeline status display in side bar of topology page : P-05- TC02
```

Comment:
I would suggest to use the Scenario and not Scenario Outline.

Reply:
Scenario Outline. should be used when we are using examples